index.js

```diff
@@ -103,13 +112,43 @@ module.exports.prototype.customAfter = field => (query, value) => {
 
   date = new Date(date);
 
+  return date;
+}
+
+module.exports.prototype.customAfter = field => (query, value) => {
+  const date = parseDate(value);
+
   if (date.toString() !== 'Invalid Date') {
     query[field] = {
       $gte: date.toISOString(),
     };
   }
 };
 
+module.exports.prototype.customBefore = field => (query, value) => {
+  const date = parseDate(value);
+
+  if (date.toString() !== 'Invalid Date') {
+    query[field] = {
+      $lt: date.toISOString(),
+    };
+  }
+};
+
+module.exports.prototype.customBetween = field => (query, value) => {
+  const [afterValue, beforeValue] = value.split('|');
```

Comment:
At the moment we need to support Node.js 4. Destructuring assignments are only supported in newer versions of Node.js (from version 6 and forward).

```js
const dates = value.split('|');
const afterValue = dates[0];
const beforeValue = dates[1];
```